### PR TITLE
[API-41173] Update POA `/decide` docs

### PR DESCRIPTION
**Pull request by tycol7**
## Summary

- Update POA `/decide` docs (dev/staging only)

## Related issue(s)

[API-41173](https://jira.devops.va.gov/browse/API-41173)

## Testing done

- [x] *New code is covered by unit tests*

## Screenshots
poa-decide-docs

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I updated unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

Wanted to delete and reorganize specs and json files, but ran into the PR line-limit rule. Any suggestions on how to move/remove these files welcome.

### CODE REVIEW

File: modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb
Comment (tycol7) [inline modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb:1]: we are covered by [request_controller_spec.rb](https://github.com/department-of-veterans-affairs/vets-api/blob/0469b95a5b71746a13b19630e4982932c6476fd9/modules/claims_api/spec/controllers/v2/veterans/power_of_attorney/request_controller_spec.rb).

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 1]
```diff
@@ -69,9 +69,9 @@
     }
   },
   "paths": {
-    "/power-of-attorney-requests/{id}/decision": {
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:72]: why the decision to remove the id from here? just curious.

File: modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb [hunk 2]
```diff
@@ -3,7 +3,9 @@
 require 'rails_helper'
 require Rails.root / 'modules/claims_api/spec/rails_helper'
 
-RSpec.describe 'ClaimsApi::V2::PowerOfAttorneyRequests::Decisions#create', :bgs, type: :request do
+# TODO: Delete this file
+# We are covered by request_controller_spec.rb
+RSpec.describe 'ClaimsApi::V2::PowerOfAttorneyRequests::Decisions#create', :bgs, :skip, type: :request do
```
Comment (FonzMP) [inline modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb:8]: this due to LoC?

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json (hunk 1 above)
Comment (tycol7) [reply modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:72]: the `{id}` in this case is the `procId` or "process id." imo, it seemed unidiomatic to include it in the URL, which implies the id belongs to a "power of attorney request"-type object. so now we explicitly send the `procId` in the body 🤷

File: modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb (hunk 2 above)
Comment (tycol7) [reply modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb:8]: Yes. I wanted to delete this file and move/rename the other files to align with the API and code directories, but ran into the line limit bot 🤖 😔

**Comment by rockwellwindsor-va** [comment]
Are the missing schemas for the responses related to the line limit too?  I'm not sure if they are 'required' or not but the docs for the other endpoints responses all seem to have them just just want to make sure.

**Comment by tycol7** [comment]
> Are the missing schemas for the responses related to the line limit too? I'm not sure if they are 'required' or not but the docs for the other endpoints responses all seem to have them just just want to make sure.

Not related to the line limit in particular, just seemed redundant imo. But happy to add them back for conformity.

**Comment by rockwellwindsor-va** [comment]
> > Are the missing schemas for the responses related to the line limit too? I'm not sure if they are 'required' or not but the docs for the other endpoints responses all seem to have them just just want to make sure.
> 
> Not related to the line limit in particular, just seemed redundant imo. But happy to add them back for conformity.

Ya I think we need to maintain conformity in this case.

**Comment by tycol7** [comment]
> Ya I think we need to maintain conformity in this case.

On a closer look, it seems other endpoints do not have 500 errors. So I think we are safe removing them here. The docs accurately reflect the endpoint's possible responses, so if we want different/more responses, I think we need another task.

**Comment by rockwellwindsor-va** [comment]
> > Ya I think we need to maintain conformity in this case.
> 
> On a closer look, it seems other endpoints do not have 500 errors documented. So I think we are safe removing those here. The docs accurately reflect the endpoint's possible responses, so if we want different/more responses, I think we need another task.

Just to be clear I am talking about the 'schema' tab on these.

Screenshot 2024-10-21 at 3 27 30 PM

**Comment by tycol7** [comment]
> Just to be clear I am talking about the 'schema' link/tab (whatever it is) missing on these after the 'Example Value'.
Oh, strange. I am not sure where those went. Taking a look - good catch!

**Comment by tycol7** [comment]
Re-added the response schemas.

schema

**Review by rockwellwindsor-va — APPROVED** [review]
See the schemas link now, docs look good.

**Pull request merged**